tests/unittests/Foundation/NSURLTests.m

```diff
@@ -206,7 +206,8 @@ void testNSURLMethod(SEL selector, NSURL* input, id argument, NSURL* expected) {
     ASSERT_OBJCEQ_MSG(fileURLString, newFileURLString, "File URLs do not match!");
 }
 
-TEST(NSURL, CheckResourceIsReachable) {
+// Reference platform has bug on some machines where currentDirectoryPath returns the incorrect path
```

Comment:
This entire test is awful and should be shot.

It manipulates global state (current directory) to do something it could simply do from the base path returned by `GetCurrentTestDirectory`.